ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,25 +69,152 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
+  var vocabularies = [
+    {
+      "vocabulary": vocabulary,
+      "terms": vocabulary.terms
     }
+  ];
+  var listOfVocabularies = {
+    "count": 1,
+    "next": null,
+    "previous": null,
+    "results": [
+      {
+        "id": 1,
+        "slug": "difficulty",
+        "name": "difficulty",
+        "description": "easy",
+        "vocabulary_type": "f",
+        "required": false,
+        "weight": 2147483647
+      }
+    ]
+  };
+  var listOfTerms = {
+    "count": 2,
+    "next": null,
+    "previous": null,
+    "results": vocabulary.terms
+  };
+  var saveVocabularyResponse;
```

Comment:
This variable used in two tests. Please create a local variable within each test instead. We want to limit global state as much as possible, especially given the complexity of our test framework. The way it is now, one test may affect another tests result which will make test failures hard to debug.